extension/src/popup/components/sendPayment/SendSettings/Slippage/index.tsx

```diff
@@ -62,8 +63,11 @@ export const SendSettingsSlippage = () => {
             );
             navigateTo(ROUTES.sendPaymentSettings);
           }}
+          validationSchema={YupObject().shape({
```

Comment:
maxing slippage to 10%
https://www.figma.com/file/BR8lyECrgUHybq7XuQSEED?node-id=1693:4012#172895146